src/node/reflection.cc

```diff
@@ -106,8 +106,12 @@ runtime::TVMRetValue ReflectionVTable::GetAttr(Object* self, const String& field
     }
   }
   if (!success) {
-    LOG(FATAL) << "AttributeError: " << self->GetTypeKey() << " object has no attributed "
-               << getter.skey;
```

Comment:
I think we should keep the LOG(FATAL). It is used in a lot of places. Instead, we should fix where ever is calling GetAttr a lot.

Reply:
I agree that LOG(FATAL) would be more debug-friendly, but I'm not sure if it is practical to fix where ever is calling GetAttr in an inefficient way, as GetAttr is used basically every where. IMHO, it doesn't hurt to make an exception use (i.e., `throw`) for this special case. Meanwhile, if we want to keep LOG(FATAL) for debugging purpose, we may consider using a macro or DLOG(FATAL).

Reply:
It seems like the issue is various python functions repeatedly calling `getattr` on TVM Objects. Could we have a separate `GetAttr` for python that does not collect a backtrace? Or we could add `HasAttr` and have the python code check that the attr exists before calling getattr.

Reply:
Removed the `GetAttr` changes as they're not necessary for this scenario with the change to `extract_tasks` -- if we see other scenarios where `__getattr__` shows up as hot we can pursue a deeper solution.